.circleci/config.yml

```diff
@@ -198,6 +217,11 @@ workflows:
             branches:
               only: master
 
+      - build_images:
+          filters:
+            branches:
+              only: packer
```

Comment:
Could you please mark from github settings branch `packer` as protected somehow? E.g. so that no one can delete branch from https://github.com/aeternity/infrastructure/branches

Reply:
@sennui Could we please protect `packer` branch ?

Reply:
Rethinking, deletion of the branch looks "harmful" to me? The next time someone push it, it will resurrect and continue working ?

If we protect it, it will also disable force pushing to it, which is better IMO than merging for this kind of branches (it's just a trigger).

Reply:
[Telepathy](https://github.com/aeternity/infrastructure/pull/61#discussion_r194372872).